src/assessment.js

```diff
@@ -50,4 +50,11 @@ var q2 = [/* your answer here */];
 //
 var q5 = function fireOnAlderaan () {
   // your code here
+
+  fireOnAlderaan.call(alderaan);
+
+  // or maybe:
+  var q5 = fireOnAlderaan.call(alderaan);
```

Comment:
Ditto. You're right that you need to call the `call` method and pass in `alderaan` as an argument, but you're not calling it on the right thing.